src/ngraph_backend_config.h

```diff
@@ -0,0 +1,57 @@
+/*******************************************************************************
+ * Copyright 2019 Intel Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+
+#ifndef NGRAPH_TF_BRIDGE_BACKEND_CONFIG_H_
+#define NGRAPH_TF_BRIDGE_BACKEND_CONFIG_H_
+
+#include <ostream>
+
+#include "ngraph_log.h"
+#include "tensorflow/core/lib/core/errors.h"
+
+using namespace std;
+
+namespace tensorflow {
+
+namespace ngraph_bridge {
+
+class BackendConfig {
+ public:
+  BackendConfig() = delete;
+  BackendConfig(string backend_name);
```

Comment:
Minor
We can make this constructor private and `BackendManager` a friend class, so no one will be able to use `BackendConfig` other than `BackendManager`. `BackendConfig` could even be a nested class in `BackendManager`

Reply:
I like the nested class idea. `Friend` - no :)

Reply:
Decided to keep as it is